src/utils/uppy-config.ts

```diff
@@ -23,9 +23,10 @@ interface UppyCompleteArgs extends UppyArgs {
   parts: AwsS3Part[];
 }
 
-export const createUppy = (): UppyType =>
+export const createUppy = (shouldAllowMultipleFiles: boolean): UppyType =>
   Uppy<Uppy.StrictTypes>({
     autoProceed: true,
+    restrictions: { maxNumberOfFiles: shouldAllowMultipleFiles ? null : 1 },
```

Comment:
kunne snudd om denne, for hvis `shouldAllowMultipleFiles=null` nå, så blir `maxNumberOfFiles=1`

Reply:
@elkebab Tenkte du variabel av type 'shouldLimitToSingleFile'  elns ?

Reply:
egentlig `{ maxNumberOfFiles: !shouldAllowMultipleFiles ? 1 : null }`, men ser nå at det blir jo samme utfall der :p 
ville bare at hvis shouldAllowMultiple er null så blir maxNumberOfFiles null også, men den er jo required uansett, så bare la det ligge :)